components/Login.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, StyleSheet } from 'react-native';
+import GoogleButton from './GoogleButton';
+import Notification from './Notification';
+import { useTranslation } from 'react-i18next';
+
+import { GoogleSignin } from '@react-native-google-signin/google-signin';
+import { googleLogin } from '../network/login-and-signup';
+import { storeUserSession } from '../libs/EncryptedStoreage';
+import CromaButton from './CromaButton';
+
+const googleSignIn = async () => {
+  try {
+    await GoogleSignin.hasPlayServices();
+    const userInfo = await GoogleSignin.signIn();
+    // this.setState({ userInfo });
+    const res = await googleLogin(userInfo);
+    await storeUserSession(
+      res.data.user.full_name,
+      res.data.user.email,
+      res.data.userToken,
+      res.data.user.avatar_url
+    );
+    // user.loggedIn = true;
+    // setUser(user);
+    // naviagteAfterLogin();
+  } catch (error) {
+    // setError(error.message);
```

Comment:
We should show errors like this to users also and send the error to server for us to debug 
https://github.com/croma-app/huehive-mobile-app/blob/7012a4971cf54586edf2ff2b33e05638081e0309/libs/Helpers.js#L20